client/app/kitchen/kitchen.component.html

```diff
@@ -1,3 +1,42 @@
-<div *ngFor="let order of orders">
-  <p>{{order | json}}</p>
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-sm-12 col-md-8 col-lg-8">
+            <table class="table table-bordered table-hover">
+                <thead>
+                <tr>
+                    <th scope="col">Table Number</th>
+                    <th scope="col">Items</th>
+                    <th scope="col">Time</th>
+                    <th scope="col">Status</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr *ngFor="let order of orders | orderBy:'status'">
+                    <th scope="row">{{order.tableId}}</th>
+                    <td>
+                        <small *ngFor="let item of order.orderItems">
+                    <td [ngClass]="{
+                            'table-warning': item.status === 'InProgress',
+                            'table-success': item.status === 'Completed'
+                          }">
+                        {{item.name}}
+                        <b>£{{item.pricePerPortion}}</b>
```

Comment:
I think dev might be using price now, instead of price per portion... might be worth checking maybe...